basics/types.md

```diff
@@ -119,42 +118,4 @@ Adam
 
 You cannot initialize arrays using only Liquid.
```

Comment:
This is technically incorrect, but I'd like to keep it that way :stuck_out_tongue: